docs/content/concepts/layer/implementation/ssp/_index.md

```diff
@@ -66,6 +66,46 @@ The figure below expresses represents the portion of the OSCAL stack as it relat
 
 {{<partialCached "note-to-developers-uuid.html" >}}
 
+### Identifier References
+
+An OSCAL SSP may contain references to information that is defined locally (e.g., in the SSP model) or externally (e.g., in a referenced profile, catalog, or component definition model).  The following lists the mechanisms through which objects in an OSCAL SSP may "link" to other OSCAL content:
+
+- **Local**: Scenario where an identifier references information in the same OSCAL instance.  This scenario supports  “compile time” enforcement of constraints since the validity of the reference can be checked as soon as the SSP is authored.
+- **Import-Profile**:  Scenario where an identifier references information in its imported profile.  This scenario can support “compile time” enforcement of constraints if there is access to the imported OSCAL profile.
+- **Leveraged-Authorizations/Links**:  In this scenario where an OSCAL SSP has leveraged-authorization(s), some of the information in the leveraged SSP may be referenced (e.g., if the leveraged SSP is also in OSCAL format).  This scenario supports “runtime” validation of constraints since SSP author may or may not have access the leveraged system SSP to validate / enforce the referential integrity.  However, the authorizing official (AO) must ultimately validate any such links and references.
+- **Component Definition (CDEF) Links**:  Scenario where an OSCAL SSP component has a reference to a component definition and/or references to a specific component within a component definition.  This scenario can support “compile time” enforcement of constraints if the component definition is available for validation.
+- **Links**: Aside from OSCAL specific links such as Leveraged-Authorizations/Link, general OSCAL links do not provide enough contextual information to know if the link target 1) is an OSCAL document and 2) what type of OSCAL model. Additionally, there is no assurance that SSP authors have access to linked OSCAL instances, so any references to content in external links cannot be validated.
+
+### Reference Constraints
+
+The following table summarizes constraints for identifier references in an OSCAL SSP:
+
+| Reference | Target Scope | Target Element | Target ID Type | Referential Constraint Description |
```

Comment:
In the table can you link to the data type definition? https://pages.nist.gov/OSCAL/reference/datatypes/

Reply:
Added